### PR TITLE
fix(deps): upgrade stylelint to 14.x

### PR DESCRIPTION
**Pull request by gyfchong**
## Why
Keep on top of latest package versions

## What
Upgraded stylelint to latest

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -18176,6 +18201,50 @@ stylelint@^13.13.1:
     v8-compile-cache "^2.3.0"
     write-file-atomic "^3.0.3"
 
+stylelint@^14.16.1:
```
Comment (HeartSquared) [inline yarn.lock:18204]: Hmm...I was expecting the old stylelint version to disappear from the yarn.lock. You might want to double check this.

Comment (gyfchong) [reply yarn.lock:18204]: Old `stylelint@13` seems to still be used by other packages.

Comment (gyfchong) [reply yarn.lock:18204]:
```stylelint@^13.13.1:
  version "13.13.1"
  resolved "https://registry.yarnpkg.com/stylelint/-/stylelint-13.13.1.tgz#fca9c9f5de7990ab26a00f167b8978f083a18f3c"
  integrity sha512-Mv+BQr5XTUrKqAXmpqm6Ddli6Ief+AiPZkRsIrAoUKFuq/ElkUh9ZMYxXD0iQNZ5ADghZKLOWz1h7hTClB7zgQ==
  dependencies:
    "@stylelint/postcss-css-in-js" "^0.37.2"
    "@stylelint/postcss-markdown" "^0.36.2"
    autoprefixer "^9.8.6"
    balanced-match "^2.0.0"
    chalk "^4.1.1"
    cosmiconfig "^7.0.0"
    debug "^4.3.1"
    execall "^2.0.0"
    fast-glob "^3.2.5"
    fastest-levenshtein "^1.0.12"
    file-entry-cache "^6.0.1"
    get-stdin "^8.0.0"
    global-modules "^2.0.0"
    globby "^11.0.3"
    globjoin "^0.1.4"
    html-tags "^3.1.0"
    ignore "^5.1.8"
    import-lazy "^4.0.0"
    imurmurhash "^0.1.4"
    known-css-properties "^0.21.0"
    lodash "^4.17.21"
    log-symbols "^4.1.0"
    mathml-tag-names "^2.1.3"
    meow "^9.0.0"
    micromatch "^4.0.4"
    normalize-selector "^0.2.0"
    postcss "^7.0.35"
    postcss-html "^0.36.0"
    postcss-less "^3.1.4"
    postcss-media-query-parser "^0.2.3"
    postcss-resolve-nested-selector "^0.1.1"
    postcss-safe-parser "^4.0.2"
    postcss-sass "^0.4.4"
    postcss-scss "^2.1.1"
    postcss-selector-parser "^6.0.5"
    postcss-syntax "^0.36.2"
    postcss-value-parser "^4.1.0"
    resolve-from "^5.0.0"
    slash "^3.0.0"
    specificity "^0.4.1"
    string-width "^4.2.2"
    strip-ansi "^6.0.0"
    style-search "^0.1.0"
    sugarss "^2.0.0"
    svg-tags "^1.0.0"
    table "^6.6.0"
    v8-compile-cache "^2.3.0"
    write-file-atomic "^3.0.3"```

Comment (HeartSquared) [reply yarn.lock:18204]: Gross 🙃

**Review by HeartSquared — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (gyfchong) [reply yarn.lock:18204]: We can't all be on top of our deps 🙃

**Pull request merged**